### ISSUE TITLE
HTTP 500 as response to key login

### ISSUE THREAD

**Issue by kjetilk** [CastagnaIT/plugin.video.netflix#956]
#### Your Environment
- Netflix add-on version: 1.11.0
- Operating system version/name: Raspbian Buster
- Device model:  RPi 4

Used Operating system:
* [ ] Android
* [ ] iOS
* [x] Linux
* [ ] OSX
* [x] Raspberry-Pi
* [ ] Windows

### Describe the bug

After successfully obtaining the `NFAuthentication.key`, I try to log in. First I get prompted for the key file, which I provide, then the PIN, which I enter, then my password (which strikes me as odd), but I give that too. 

#### Expected behavior

I would be expected to log in.

#### Actual behavior

I get an error: 

``` 
500 Server Error: Internal Server Error for url: https://www.netflix.com/api/shakti/v0c29af6f/profilehub
```

#### Steps to reproduce the behavior

1. Get key
2. Attempt log in
3. See it fails 

(I realize this is probably not useful, but that's really all that happens)

### Debug log

The debug log can be found from this link:
https://pastebin.com/q7YVwNNS
I hope I put on the correct verbose mode :-)

#### Installation
* [ ] I'm using other Netflix Repo 
* [ ] I'm using a different source 

#### Other information

It seems the problem is in Netflix' end, but there are probably some assumptions that are violated here.

**Comment by CastagnaIT** [comment]
Please try generate the file key again with this updated version:
[NFAuthenticationKey_Linux_t1.zip](https://github.com/CastagnaIT/plugin.video.netflix/files/5629552/NFAuthenticationKey_Linux_t1.zip)

**Comment by kjetilk** [comment]
OK, great, thanks! I'll check if it works!

**Comment by kjetilk** [comment]
Yes, now it worked! :+1: 

However, the reason it worked now is probably something more embarrassing...: Somehow, my setup of my Norwegian QWERTY keyboard had not taken effect, and one character in the password was mapped to the wrong key... I got the key OK because I obtained that on a different machine, but it failed on the RPi running Kodi. 

Perhaps it should be a better error message in such cases?
